src/nav/NavBarBackButton.js

```diff
@@ -18,13 +20,16 @@ import * as NavigationService from './NavigationService';
  * https://material.io/design/navigation/understanding-navigation.html
  */
 // TODO: on iOS, show the right icon for a back button
-// TODO: on iOS, give the right label for a back button
 export default function NavBarBackButton(props: {| +color?: string |}) {
+  const _ = useContext(TranslationContext);
+
+  const accessibilityLabel = Platform.OS === 'ios' ? _('Back') : _('Navigate up');
```

Comment:
nit: Let's inline this instead of putting it in a new variable. So, down below,

```
accessibilityLabel={Platform.OS === 'ios' ? _('Back') : _('Navigate up')}
```